cmd/sovereignnode/incomingHeader/incomingHeaderProcessor_test.go

```diff
@@ -60,9 +67,9 @@ func createIncomingHeadersWithIncrementalRound(numRounds uint64) []sovereign.Inc
 			},
 			IncomingEvents: []*transaction.Event{
 				{
-					Topics:     [][]byte{[]byte("addr"), []byte("tokenID1"), []byte("nonce1"), []byte("val1")},
+					Topics:     [][]byte{[]byte("endpoint"), []byte("addr"), []byte("tokenID1"), []byte("nonce1"), tokenData},
```

Comment:
Let's please stick to only one term for this "endpoint" and use everywhere topicID